agile_bdd/spec/features/websearch_spec_test.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+require 'websearch'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.